src/amcl/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
There should be a "localization" directory under which we can put multiple implementations of localization. So, I'm thinking:<root>/localization/amcl

Reply:
I agree.